test/harry/main/org/apache/cassandra/harry/util/StringUtils.java

```diff
@@ -35,7 +35,7 @@ public static String escapeControlChars(String input)
         for (int i = 0; i < input.length(); i++)
         {
             char c = input.charAt(i);
-            if (Character.isISOControl(c))
+            if (Character.isISOControl(c) && c != '\n')
```

Comment:
you don't see it in this patch but in `BEGIN BATCH` the statements are multi line, which got escaped in the logs